### PR TITLE
Add gpt2 to unittest; fix regression

### PR DESCRIPTION
**Pull request by BowenBao**
I'm slightly not sure about if we should design the api with `opset_version` as positional argument. However, the scope of the PR is only to fix the regression, under the assumption that `opset_version` is a positional argument.

### CODE REVIEW

**Comment by BowenBao** [comment]
cc @titaiwangms @justinchuby @wschin

File: test/onnx/test_fx_to_onnx_with_onnxruntime.py [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
 
 import onnx_test_common
+import onnxruntime  # type: ignore[import]
 import torch
 from torch import nn
 from torch.nn import functional as F
+from torch.onnx._internal import fx as fx_onnx
 from torch.testing._internal import common_utils
+from torch.utils import _pytree as pytree
+from transformers import AutoModel, AutoTokenizer  # type: ignore[import]
```
Comment (justinchuby) [inline test/onnx/test_fx_to_onnx_with_onnxruntime.py:14]: import modules only?

File: test/onnx/test_fx_to_onnx_with_onnxruntime.py [hunk 2]
```diff
@@ -54,6 +75,26 @@ def forward(self, tensor_x: torch.Tensor):
...
+        input_ids = inputs["input_ids"]
+        attention_mask = inputs["attention_mask"]
+
+        onnx_model = fx_onnx.export_without_kwargs(
+            model, self.opset_version, **inputs, use_binary_format=True
+        )
+
+        ref_outputs, _ = pytree.tree_flatten(model(**inputs, return_dict=False))
+        ort_outputs = self._run_ort(onnx_model, (input_ids, attention_mask))
+        for ref_output, ort_output in zip(ref_outputs, ort_outputs):
```
Comment (justinchuby) [inline test/onnx/test_fx_to_onnx_with_onnxruntime.py:95]: also assert nonempty and equal length?

File: torch/onnx/_internal/fx/export.py [hunk 3]
```diff
@@ -470,6 +470,7 @@ def export(
 
 def export_without_kwargs(
     fn: Union[torch.nn.Module, Callable],
+    opset_version,
```
Comment (justinchuby) [inline torch/onnx/_internal/fx/export.py:473]: Possible to make it optional and keyword only? so

fn, *args, ...

**Comment by BowenBao** [comment]
> I could be missing out some discussion. Why are we considering not have opset_version api? Which version should we stick to?

There is no discussion yet afaik. There could be many ways how opset_version can be specified (positional, optional, config etc), but that's fine and can be consolidated later. I wanted to limit this PR to be very simple and just enable gpt2 unittest with the current api.

**Review by titaiwangms — APPROVED** [review]

File: torch/onnx/_internal/fx/export.py (hunk 3 above)
Comment (BowenBao) [reply torch/onnx/_internal/fx/export.py:473]: out of scope for this PR, but I'd love to hear your opinion on this. Which would you prefer?

* optional, default to some default (same behavior as current exporter)
* optional, default to latest
* required
* specified via a separate 'config.py' file (what dynamo and inductor does)
* not configurable

**Pull request merged**